lib/sycamore/sycamore/connectors/duckdb/duckdb_writer.py

```diff
@@ -59,7 +56,9 @@ def compatible_with(self, other: BaseDBWriter.TargetParams) -> bool:
 class DuckDBClient(BaseDBWriter.Client):
     @requires_modules("duckdb", extra="duckdb")
     def __init__(self, client_params: DuckDBWriterClientParams):
-        pass
+        if not client_params.db_url:
+            raise ValueError(f"Must provide valid disk location. Location Specified: {client_params.db_url}")
```

Comment:
"Must provide valid disk location" feels like it could be misleading since this errors if it's "" or None or something

Reply:
Changed.